tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -7,6 +7,22 @@ drop_packets/test_drop_counters.py::test_loopback_filter:
   skip:
     reason: "SONiC can't enable loop-back filter feature"
 
+platform_tests/api/test_thermal.py::TestThermalApi::test_get_model:
+   # Hardware components that we use for our sensors does not have IDPROM to store model and serial number details. 
+   # Due to this Cisco currently does not expose serial and model number under sysfs path
+   skip:
+     reason: "test_get_model is not supported in cisco platform"
+     conditions:
+        - asic_type=="cisco-8000"
+
+ platform_tests/api/test_thermal.py::TestThermalApi::test_get_serial:
+   # Hardware components that we use for our sensors does not have IDPROM to store model and serial number details.
+   # Due to this Cisco currently does not expose serial and model number under sysfs path
+   skip:
+     reason: "test_get_serial is not supported in cisco platform"
+     conditions:
+        - asic_type=="cisco-8000"
```

Comment:
Please fix the indention issue. Otherwise, the mark definition file can't be parsed.